<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqQueryClauseWidget</class>
 <widget class="QWidget" name="pqQueryClauseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="criteria"/>
     </item>
     <item>
      <widget class="QComboBox" name="condition">
       <item>
        <property name="text">
         <string>is</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is between</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is one of</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is &gt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is &lt;=</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="valueStackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="singleValue">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="range">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value_min"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>and</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="value_max"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="location">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value_x"/>
         </item>
         <item>
          <widget class="QLineEdit" name="value_y"/>
         </item>
         <item>
          <widget class="QLineEdit" name="value_z"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="multiBlock">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="value_block"/>
         </item>
         <item>
          <widget class="QToolButton" name="showCompositeTree">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remove">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
